### DISCUSSION TITLE
remove value providers

### DISCUSSION THREAD

**Discussion by robertmusialek** [airbnb/lottie-ios#2390] [Q&A]
Hey,

Is there a way to remove `ValueProvider` added to the animation? 

Previously trick like:
```
animationView.animation = animationView.animation
```
used to work, but this recently changed that value providers are reapplied when rebuilding animation layer

any input how to solve this is welcome

cheers,
R

**Comment by batanus** [comment]
It was added in this PR: 
- https://github.com/airbnb/lottie-ios/pull/2474
